pkg/types/nutanix/metadata.go

```diff
@@ -0,0 +1,15 @@
+package nutanix
+
+// Metadata contains Nutanix metadata (e.g. for uninstalling the cluster).
+type Metadata struct {
+	// PrismCentral is the domain name or IP address of the Prism Central.
+	PrismCentral string `json:"prismCentral"`
+	// Username is the name of the user to use to connect to the Prism Central.
+	Username string `json:"username"`
+	// Password is the password for the user to use to connect to the Prism Central.
+	Password string `json:"password"`
+	// Port is the port used to connect to the Prism Central.
+	Port string `json:"port"`
+	// Disable certificate checking when connecting to Prism Central.
+	Insecure bool `json:"insecure"`
```

Comment:
We should not be supporting insecure connections. Let's not repeat the mistake made for vSphere.

Reply:
For context. This is still https. We just allow disabling cert checking due to self signed certs.

Reply:
We should not be allowing insecure to support self-signed certs. We should be requiring the user to provide the CA bundle to be able to trust the self-signed certs.

Reply:
+1 to staebler's guidance on requiring a CA bundle to trust self-signed certs.  The team is having discussions on how the CA bundle can be obtained.

Reply:
@thunderboltsid we will just have to doc download/import of certs like here: https://docs.openshift.com/container-platform/4.9/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-adding-vcenter-root-certificates_installing-vsphere-installer-provisioned